### ISSUE TITLE
Polls 1 of 3 gmails evry 10 instead of set 60 mins...

### ISSUE THREAD

**Issue by Cam-Gibb** [k9mail/k-9#1235]
### Expected behaviour

Tell us what should happen

Should only poll every hr but polls every 10 mins. 2 gmails and 1 other. Only does this with 1 big use gmail.
### Actual behaviour

Tell us what happens instead

Polls evry 10 mins instead of set hrly.
### Steps to reproduce
1. Disabled in Cell apps settings, cleared cache.
2. Restarted K9, changed polling to 30.
3. Repeated all above, set poll to hr.
### Environment

K-9 Mail version: Latest Googleplay K9 update.

Android version: 4.1.2 Sony Xperia LT28i

Account type (IMAP, POP3, WebDAV/Exchange):

POP3.

Didn't do this before for many mths, until about few mths ago. I suspect after auto-updating via Playstore app.

**Comment by philipwhiuk** [comment]
Just to clarify, you have 3 accounts, all POP3, two of which are Gmail-based and one which isn't. And one of the Gmail accounts gets polled every 10 minutes instead of every hour?

**Comment by Cam-Gibb** [comment]
Yes this one. About 1/2 the time. I just watched it with notifications bar pulled down, when I renabled my internet connection. It checked this email 1st. Showed new emails for this address while it also showed polling the other 2 addresses. When it was done polling, the note bar note showing new emails vanished. It started doing this week or 2 ago.

Cam Gibb.

On 31 March, 2016 5:01:12 AM EDT, Philip <notifications@github.com> wrote:
>Just to clarify, you have 3 accounts, all POP3, two of which are
>Gmail-based and one which isn't. And one of the Gmail accounts gets
>polled every 10 minutes instead of every hour?
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/k9mail/k-9/issues/1235#issuecomment-203833690

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by philipwhiuk** [comment]
"When it was done polling, the note bar note showing new emails vanished. It started doing this week or 2 ago."

That's clearly a separate issue, probably related to #834 I guess... After/during the design overhaul there is discussion around doing a "New Emails" aggregated virtual folder which will solve this issue. No timeline for that though.

**Comment by Cam-Gibb** [comment]
Thx. Your reply was polled automatically just now, 10 mins since last poll. Even though it shown next poll 50 mins. I was browsing FB with Google Chrome. This time your 2nd reply notification stayed in android upper notifications bar.

Should I try older version apk. ??

Where can I find it. ??

Thx from Cam Gibb.

On 31 March, 2016 5:34:47 AM EDT, Philip <notifications@github.com> wrote:
>"When it was done polling, the note bar note showing new emails
>vanished. It started doing this week or 2 ago."
>
>That's clearly a separate issue, probably related to #834 I guess...
>After/during the design overhaul there is discussion around doing a
>"New Emails" aggregated virtual folder which will solve this issue. No
>timeline for that though.
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/k9mail/k-9/issues/1235#issuecomment-203850631

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by Cam-Gibb** [comment]
I installed 2 mths back, version 5.008, over the newest that came out mths before. Dunno if it was 009 or 010. To see if new constant frequent polling issue ceased. It didn't. Ended up with duplicate folders. Tolerated for mths. Eventually found mention of ridding dup'd folders with "repopulate folders". Dup'd folders gone as of few days ago. Still sporadically polls every minute or mins or 15 or 30 despite set for 60. Not a huge deal I guess except lags system and screen response at times. Should I can I try version older then 008. ?? When first started usung it about 2-3 yrs ago it had no issues with this cell that came out around 2011.
